packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -314,6 +314,8 @@ export default ({
       order.extraAttributes.authorisationUrl
     ) {
       return order
+    } else if (order.state === 'FAILED') {
```

Comment:
we should typed `order.state` and use enum for such and get rid of hardcoded values